docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,142 @@
+Policy for Unofficial and Unmaintained Projects
+===============================================
+
+This policy describes a process where we take actions against unmaintained_ and unofficial_ forks of project documentation.
+
+
+Rationale
+---------
+
+Documentation projects may be kept online indefinitely, even though a newer version of the same project exists elsewhere.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in search results,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow a reporter to request the *delisting* of forks that are old and outdated.
+
+
+High level overview
+~~~~~~~~~~~~~~~~~~~
+
+The process at a high level looks like:
+
+* A reporter emails us about a project they think is outdated and unofficial
+* An RTD team member evaluates it to make sure it's outdated and unofficial, according to this policy
+* We delist this project from search results and send an email to owners of the RTD project
+* If a project owner objects, we evaluate their evidence and make a final decision
+
+
+Definitions
+-----------
+
+
+Source project
+~~~~~~~~~~~~~~
+
+For a given documentation project to be assessed, the *source project* comes into question. This is defined as:
+
+* The origin project from which the documentation project's repository was forked (if forked).
+* The origin project from which the documentation project was copied (if not forked but evidently copied).
+* The original project which the documentation project refers as the subject of documentation.
+* Any other project that is understood by the core team to be a the *source project*.
```

Comment:
This is an added definition @agjohnson @ericholscher 

It could potentially be simplified. Reading through the definitions I find that this is a nice intro to talking about what "unofficial" means.

Reply:
I'd say we could remove the "Source project" definition completely. I think it adds more confusion and clarification. Currently, we are only using "source project" in the email template, that can be just replaced by "official project" which is clearer in my opinion.

Reply:
It's gone, by popular demand :smile: